File: bj/10250.py
import sys
t = int(sys.stdin.readline().strip())
for i in range(t):
    h, w, n = map(int, sys.stdin.readline().strip().split())
    front = f"{(n-1)%h+1}"
    back = f"0{(n-1)//h+1}" if ((n-1)//h+1) < 10 else f"{(n-1)//h+1}"
    print(front+back)

File: bj/10250.cpp
#include<iostream>
#include <string>
using namespace std;
int main(){
    int t, h, w, n;
    cin >> t;
    for(int i = 0; i < t; i++){
        cin >> h >> w >> n;
        string front = to_string((n-1)%h+1);
        string back = int((n-1)/h)+1 < 10 ? "0" + to_string(int((n-1)/h)+1) : to_string(int((n-1)/h)+1);
        cout << front << back << endl;
    }
    return 0;
}